### DISCUSSION TITLE
Proposal: The ability to change a ship's weapons

### DISCUSSION THREAD

**Discussion by SMG001** [bcholmes/StarTrek2d20#131] [Ideas]
As Felderburg has stated in an earlier discussion, we can't really use the spinal lance weapons with any of the current spaceframes on the app. As such, I'd like to suggest a button (such as the "allow cross-species talents" button in the Character Creator) that allows us to change some parts of the spaceframe as we build it. Such as the weapons (Imagine a constitution class spaceframe with phaser arrays and quantum torpedoes).

**Comment by Felderburg** [comment]
Per my note in #130, I tried using standard spaceframe builds (in 240X years to ensure Spinal Lance availability) and didn't see any sort of weapon selection screen, nor a way to change weapons once the build was finished. It would indeed be nice if this was available, especially given the (probably STO-motivated) expansion of weapon types in the Utopia Planitia book.
